tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Couchbase/ClusterNodeIntegrationTer.cs

```diff
@@ -44,8 +44,10 @@ public class ClusterNodeIntegrationTer
         /// <typeparam name="TOperation">Type of the operation</typeparam>
         /// <returns>Calltarget state value</returns>
         internal static CallTargetState OnMethodBegin<TTarget, TOperation>(TTarget instance, TOperation operation)
+            where TTarget : IClusterNode
         {
-            return CouchbaseCommon.CommonOnMethodBeginV3(operation);
+            var normalizedSeedNodes = CouchbaseCommon.GetNormalizedSeedNodesFromConnectionString(instance.Context.ClusterOptions.ConnectionStringValue);
```

Comment:
nitpicking but if the integration is disabled we do that for nothing. Also doing it here introduces duplication.
Could we just pass a `IClusterNode` to `CommonOnMethodBeginV3` and call `CouchbaseCommon.GetNormalizedSeedNodesFromConnectionString` after checking the integration is enabled?